pkg/apisix/cluster.go

```diff
@@ -279,6 +283,7 @@ func (c *cluster) syncCacheOnce(ctx context.Context) (bool, error) {
 			)
 		}
 	}
+	c.metricsCollector.IncrSyncSuccess("cache")
```

Comment:
We may use another counter to record the cache synchronization, as below you use this counter to record the resource sync.

Reply:
It's decided to use a new counter to record this situation:
HELP apisix_ingress_controller_cache_sync_operation Number of cache sync operation  
TYPE apisix_ingress_controller_cache_sync_operation counter  
Labels:

* controller_class
* controller_namespace
* controller_pod
* resource
* state (success, error)